pymc3/sampling.py

```diff
@@ -1043,6 +1047,9 @@ def sample_posterior_predictive(trace, samples=None, model=None, vars=None, size
     vars : iterable
```

Comment:
You should add a note saying that it's deprecated.

Reply:
Oh, are you suggesting I should deprecate this? I wasn't going to go that far -- just add `varnames` as an easier-to-remember alternative. OK, will do.

Reply:
I think that supporting two parametrizations is just redundant and I think that we should opt for one or the other. @twiecki, @junpenglao, what do you think?

Reply:
I figured you would prefer I not break old code, including example notebooks, that use the old `vars` parameter.
I think `varnames` is better because it preserves the abstraction around the model object, but I could be wrong.